legacy_app.py

```
@@ -16,18 +16,17 @@
 
-import dotenv
-from fastapi import APIRouter, Body, FastAPI, HTTPException
+from fastapi import APIRouter, Body, HTTPException
 from fastapi.responses import JSONResponse
 from pydantic import (
     BaseModel,
     ValidationError,
 )
-from settings import get_runtime_env_name
-
 from app.application_metadata import build_application_metadata
```

Flake8 (7.3.0)

[error] 'app.application_metadata.build_application_metadata' imported but unused

(F401)